contrib/backporting/submit-backport

```diff
@@ -18,6 +18,15 @@ DIR=$(dirname $(readlink -ne $BASH_SOURCE))
 source $DIR/../release/lib/common.sh
 source $DIR/common.sh
 
+require_linux
```

Comment:
Seems like the merge conflict resolution pulled in this line here accidentally. It should be removed, as the `require_linux` function has not been backported to 1.8

Reply:
@nathanjsweet I think it's probably the easiest if you backported commit https://github.com/cilium/cilium/commit/05eabc71a8525fffba8977eabdfd6e763dcd08c7 as well (not the full PR, just the commit). 

I'll do the same for the v1.7 and v1.9 branch.